### DISCUSSION TITLE
Getting Developer keys

### DISCUSSION THREAD

**Discussion by Pavangj959** [community/community#24620] [Apps, API and Webhooks]
<p>Hi,<br>
I am trying to get the developer keys in the github<br>
Any suggestions how to get the developer keys and also, is the developer keys and personal access tokens are same?<br>
I am so confused, Any help would be appreciated.</p>

**Comment by jfullstackdev** [comment]
<aside class="quote no-group" data-username="Pavangj959" data-post="1" data-topic="219355">
<div class="title">
<div class="quote-controls"></div>
 Pavangj959:</div>
<blockquote>
<p>personal access tokens</p>
</blockquote>
</aside>
<p>check this out,<br>
<a href="https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token" rel="noopener nofollow ugc">https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token</a></p>

**Comment by byrneh** [comment, marked as answer]
<p>It’s probably just a naming confusion you have.<br>
There are personal access tokens a user can add to their account under settings&gt;developer settings&gt;personal access tokens.<br>
There are also SSH keys a user can add to their account under settings &gt; SSH and GPG keys.</p>
<p>It is also possible to add a deploy key to a repository, this SSH key is not associated with a user account but to the repository itself.</p>
<p>There is not something called a ‘developer key’.</p>

**Comment by Pavangj959** [comment]
<p>Hi all,<br>
I am trying to find the developer key of the stackoverflow to use their api, but I didn’t able to find it. If their is no developer key, then what is the alternative for that i.e., how can I use the stackoverflow api then?<br>
Any help or link regarding this, would be appreciated.</p>

**Comment by jfullstackdev** [comment]
<p>if I’m getting it correctly, this is the one you are looking for,</p>
<p><a href="https://api.stackexchange.com/docs">https://api.stackexchange.com/docs</a></p>
<p>it says,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/a/faa6bb741516eb24cde9076feef8435015460d7f.png" data-download-href="/uploads/short-url/zLmAz9eRxxbxOCkFffyGG3E4MlF.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181085692-6c4b4d40-96f0-4afd-8314-b7086ceb9dc9.png" alt="image" data-base62-sha1="zLmAz9eRxxbxOCkFffyGG3E4MlF" width="690" height="206" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/a/faa6bb741516eb24cde9076feef8435015460d7f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">857×257 10.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
